README.md

```diff
@@ -29,7 +29,7 @@ Hackintosh:
 
 ## Build
 
-The following kernel version(s) can be successfully built at the moment:
+The following kernel version(s) has been successfully built at the moment:
```

Comment:
I'd like to keep can be for the plurality in the future

Reply:
I think it depends on whether you are stressing your progress ("has been") or encourage people to try it ("can") but I guess it don't matter.

Reply:
Right